### PR TITLE
[SKIP CI] Tools: Tune: SRC: Add to coefficients data doxygen cond comments

### PR DESCRIPTION
**Pull request by singalsu**
There's a small fix included in second commit for earlier unnoticed error.

### CODE REVIEW

File: tools/tune/src/src_export_coef.m [hunk 1]
```diff
@@ -46,6 +46,7 @@
 	fprintf(fh, ' *\n');
```
Comment (singalsu) [inline tools/tune/src/src_export_coef.m:46]: typo in commit message "beging"

File: tools/tune/src/src_export_coef.m [hunk 2]
```diff
@@ -46,6 +46,7 @@
...
 	fprintf(fh, ' */\n');
 	fprintf(fh, '\n');
+	fprintf(fh, '/** \\cond GENERATED_BY_TOOLS_TUNE_SRC */\n');
```
Comment (marc-hb) [inline tools/tune/src/src_export_coef.m:49]:
```suggestion
	fprintf(fh, '%s\n', /** \cond GENERATED_BY_TOOLS_TUNE_SRC */');
```

File: tools/tune/src/src_export_table_2s.m [hunk 3]
```diff
@@ -48,6 +48,8 @@
 fprintf(fh, ' *\n');
 fprintf(fh, ' */\n');
 fprintf(fh, '\n');
+fprintf(fh, '/** \\cond GENERATED_BY_TOOLS_TUNE_SRC */\n');
+fprintf(fh, '\n');
```
Comment (marc-hb) [inline tools/tune/src/src_export_table_2s.m:52]:
```suggestion
fprintf(fh, '%s\n\n', '/** \cond GENERATED_BY_TOOLS_TUNE_SRC */');
```

**Review by marc-hb — COMMENTED** [review]
doxygen LGTM, no idea about the gain commit.

File: tools/tune/src/src_export_coef.m (hunk 2 above)
Comment (singalsu) [reply tools/tune/src/src_export_coef.m:49]: Your suggestion works, you can run the line from command prompt like this:

```
>> fprintf(1, '%s\n', '/** \cond GENERATED_BY_TOOLS_TUNE_SRC */');
/** \cond GENERATED_BY_TOOLS_TUNE_SRC */
```

But I prefer the simpler \ to escape the backslash character. Your way feels like a hazard, doesn't it?

File: tools/tune/src/src_export_table_2s.m (hunk 3 above)
Comment (singalsu) [reply tools/tune/src/src_export_table_2s.m:52]: Yep, I agree

**Comment by singalsu** [comment]
> doxygen LGTM, no idea about the gain commit.

Thanks, the -1 dB gain add allows the previous SRC conversions generator scripts to run without the error and produce the same coefficients values as current (except comment additions for doxygen). I noticed the error when testing this change for doxygen. I've changed for IPC4 firmware compatibility the overall gain to 0 dB per our internal discussion. Probably the other coefficients should be the same 0 dB attenuating but I didn't want to update everything now, just fix the run error and produce the same data as earlier.

**Review by lgirdwood — COMMENTED** [review]
@singalsu I'm assuming you will do this for other coefficients once you and @marc-hb align ?

File: tools/tune/src/src_export_table_2s.m (hunk 3 above)
Comment (marc-hb) [reply tools/tune/src/src_export_table_2s.m]: > Yep, I agree. I'm not sure what above "Commit suggestion" button will do (create one more commit?) so I will just manually update this PR.

Yes the "commit suggestion" is compatible only with "regular" Github usage: single commit per PR + fixup commits. It's not really compatible with our force-push usage.

File: tools/tune/src/src_export_coef.m (hunk 2 above)
Comment (marc-hb) [reply tools/tune/src/src_export_coef.m:49]: > Your way feels like a hazard, doesn't it?

? I think confusing the format string and the message is the hazard actually, it's just one step away from `warning: format not a string literal`. I think it's "Good Practice" to always keep them separate and never think about it.

Doesn't matter here, it's not like you can write memory corruption bugs in matlab...

**Review by marc-hb — APPROVED** [review]

**Comment by singalsu** [comment]
> @singalsu I'm assuming you will do this for other coefficients once you and @marc-hb align ?

Only the ipc4 coefficients set that Marc fixed caused problems with Doxygen. So, there's no need to add these comments to other SRC component files. I could do that later if we decide to change them also to 0 dB gain that would justify a patch for them.

**Comment by marc-hb** [comment]
I think the other question is: "should the other ones be in the Doxygen output by default?". Note the `\cond` directive allows including stuff with a local, one-line configuration change.

**Review by lgirdwood — APPROVED** [review]

**Pull request merged**